services/core/MasterDriverAgent/master_driver/driver.py

```diff
@@ -258,10 +258,12 @@ def periodic_read(self, now):
 
         utcnow = utils.get_aware_utc_now()
         utcnow_string = utils.format_timestamp(utcnow)
+        sync_timestamp = (now - datetime.timedelta(seconds=self.time_slot_offset)).isoformat()
```

Comment:
We need to use utils.format_timestamp to convert to a string. That will give the VOLTTRON standard date/time format. This will also make sure that the tz info is not lost. The corresponding parser is optimized for this format.

Reply:
Change committed.